pkg/cloud/azure/services/virtualmachines/virtualmachines.go

```diff
@@ -126,6 +126,40 @@ func (s *Service) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
 	if compute.OperatingSystemTypes(vmSpec.OSDisk.OSType) == compute.Windows {
 		osProfile.WindowsConfiguration = &compute.WindowsConfiguration{
 			EnableAutomaticUpdates: to.BoolPtr(false),
+			AdditionalUnattendContent: &[]compute.AdditionalUnattendContent{
+				{
+					PassName:      "OobeSystem",
+					ComponentName: "Microsoft-Windows-Shell-Setup",
+					SettingName:   "AutoLogon",
+					Content: to.StringPtr(`<AutoLogon>
+							<Username>` + *osProfile.AdminUsername + `</Username>
```

Comment:
what indentation convention is this following?
Can this leave into a variable?

Reply:
I'm not sure this is what you mean:
This is xml with 1 tab indentation. 

Could you elaborate on the second question?

Reply:
We could take the XML out of here and make it a constant with the Username and Password as fmt variables and then here we would just do 

```
Content: to.StringPtr(autoLogonFormat, *osProfile.AdminUsername, *osProfile.AdminPassword),
```

Can we do the same with the `FirstLogonCommands` as well? Move to a constant at the top of this file? That one doesn't look like it needs to be a format string though